commit 13f13416eba2beae2f83de32352ca6dd5afa6f6d
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Thu Sep 27 23:08:40 2018 +0800

    fix: compile errors under networkmanager 1.14+
    
    Add new empty functions to pass the build
    
    Change-Id: I67e8b2ff0d797b95531efc396964cd78efb0039a

diff --git a/network/nm_setting_beans_extend.go b/network/nm_setting_beans_extend.go
index 3be4b7e3..d42b9751 100644
--- a/network/nm_setting_beans_extend.go
+++ b/network/nm_setting_beans_extend.go
@@ -20,6 +20,16 @@
 package network
 
 // Empty functions to make compile success
+func getSetting6LowpanAvailableKeys(data connectionData) (keys []string) {
+	return
+}
+func getSetting6LowpanAvailableValues(data connectionData, key string) (values []kvalue) {
+	return
+}
+func checkSetting6LowpanValues(data connectionData) (err sectionErrors) {
+	return
+}
+
 func getSettingAdslAvailableKeys(data connectionData) (keys []string) {
 	return
 }
@@ -80,6 +90,16 @@ func checkSettingDcbValues(data connectionData) (err sectionErrors) {
 	return
 }
 
+func getSettingEthtoolAvailableKeys(data connectionData) (keys []string) {
+	return
+}
+func getSettingEthtoolAvailableValues(data connectionData, key string) (values []kvalue) {
+	return
+}
+func checkSettingEthtoolValues(data connectionData) (err sectionErrors) {
+	return
+}
+
 func getSettingGenericAvailableKeys(data connectionData) (keys []string) {
 	return
 }
@@ -130,6 +150,16 @@ func checkSettingMacvlanValues(data connectionData) (err sectionErrors) {
 	return
 }
 
+func getSettingMatchAvailableKeys(data connectionData) (keys []string) {
+	return
+}
+func getSettingMatchAvailableValues(data connectionData, key string) (values []kvalue) {
+	return
+}
+func checkSettingMatchValues(data connectionData) (err sectionErrors) {
+	return
+}
+
 func getSettingOlpcMeshAvailableKeys(data connectionData) (keys []string) {
 	return
 }
@@ -190,6 +220,16 @@ func checkSettingProxyValues(data connectionData) (err sectionErrors) {
 	return
 }
 
+func getSettingSriovAvailableKeys(data connectionData) (keys []string) {
+	return
+}
+func getSettingSriovAvailableValues(data connectionData, key string) (values []kvalue) {
+	return
+}
+func checkSettingSriovValues(data connectionData) (err sectionErrors) {
+	return
+}
+
 func getSettingTCConfigAvailableKeys(data connectionData) (keys []string) {
 	return
 }
@@ -260,6 +300,16 @@ func checkSettingWimaxValues(data connectionData) (err sectionErrors) {
 	return
 }
 
+func getSettingWpanAvailableKeys(data connectionData) (keys []string) {
+	return
+}
+func getSettingWpanAvailableValues(data connectionData, key string) (values []kvalue) {
+	return
+}
+func checkSettingWpanValues(data connectionData) (err sectionErrors) {
+	return
+}
+
 func setSettingWirelessAssignedMacAddressJSON(data connectionData, valueJSON string) (err error) {
 	return setSettingKeyJSON(data, "802-11-wireless", "assigned-mac-address", valueJSON, ktypeWrapperString)
 }
